ShareTheTable/View Controllers/Profile Making/Add Photos/AddPhotosViewController.m

```diff
@@ -0,0 +1,137 @@
+    SceneDelegate *sceneDelegate = (SceneDelegate *)[UIApplication sharedApplication].connectedScenes.allObjects.firstObject.delegate;
+    
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+    
+    CreateAccountViewController *navViewController = [storyboard instantiateViewControllerWithIdentifier:@"AccountNav"];
+    sceneDelegate.window.rootViewController = navViewController;
+    
+    // TODO: DELETE ACCOUNT MADE
+}
+
+- (IBAction)tapSelectPhotos:(id)sender {
+    // self.imageViews = [[NSMutableArray alloc] init];
+    PHPickerConfiguration *config = [[PHPickerConfiguration alloc] init];
+    config.selectionLimit = 5;
+    config.filter = [PHPickerFilter imagesFilter];
+
+    PHPickerViewController *pickerViewController = [[PHPickerViewController alloc] initWithConfiguration:config];
+    pickerViewController.delegate = self;
+    [self presentViewController:pickerViewController animated:YES completion:nil];
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.imageViews = [[NSMutableArray alloc] init];
+}
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
+
+-(UIImageView*)newImageViewForImage:(UIImage*)image {
+    UIImageView *imageView = [[UIImageView alloc] init];
+    imageView.contentMode = UIViewContentModeScaleAspectFit;
+    imageView.backgroundColor = [UIColor blackColor];
+    imageView.image = image;
+    return imageView;
+}
+
+- (UIImage *)resizeImage:(UIImage *)image withSize:(CGSize)size {
+    UIImageView *resizeImageView = [[UIImageView alloc] initWithFrame:CGRectMake(0, 0, size.width, size.height)];
+    
+    resizeImageView.contentMode = UIViewContentModeScaleAspectFill;
+    resizeImageView.image = image;
+    
+    UIGraphicsBeginImageContext(size);
+    [resizeImageView.layer renderInContext:UIGraphicsGetCurrentContext()];
+    UIImage *newImage = UIGraphicsGetImageFromCurrentImageContext();
+    UIGraphicsEndImageContext();
+    
+    return newImage;
+}
+
+- (void)picker:(nonnull PHPickerViewController *)picker didFinishPicking:(nonnull NSArray<PHPickerResult *> *)results {
+    
+    [picker dismissViewControllerAnimated:YES completion:nil];
+
+    for (PHPickerResult *result in results) {
+        NSLog(@"result: %@", result);
+        NSLog(@"%@", result.assetIdentifier);
+        NSLog(@"%@", result.itemProvider);
+        
+        // Get UIImage
+        [result.itemProvider loadObjectOfClass:[UIImage class] completionHandler:^(__kindof id<NSItemProviderReading>  _Nullable object, NSError * _Nullable error) {
+
+            NSLog(@"object: %@, error: %@", object, error);
```

Comment:
This is great for debugging and checking that everything works but you shouldn't commit logging in your production code. 
For future pull requests, make sure to remove all logging.